app/models/application_record.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def formatted_time
```

Comment:
This is great!  Consider adding a test for this to confirm it returns what we want.